ctat_mutations

```diff
@@ -696,31 +398,90 @@ class RnaseqSnp:
+                             genome_fa,
+                             vcf_file,
+                             gatk_path,
+                             picard_path):
 
 
-    def func_do_variant_calling_use_vcf(self, args_call,
+         # Commands which will be returned
+        lcmd_gatk_variants_commands = []
+
+        # Perform recalibration
+        align_refine_cmd, str_return_bam, str_return_bai = self.RNA_refine_mapping(args_call,
+                                                                                   str_align_file,
+                                                                                   str_unique_id, str_project_dir,
+                                                                                   str_tmp_dir,
+                                                                                   genome_fa,vcf_file,
+                                                                                   gatk_path,picard_path)
+        lcmd_gatk_variants_commands.extend(align_refine_cmd)
+   
+        # Files
+        str_variants_file = os.path.join(str_project_dir, "variants.vcf")
+
+        # Create depth file
+        if args_call.f_calculate_base_coverage:
+            str_depth_compressed_file = os.path.basename(args_call.str_out_dir) + ".depth"
+            str_depth_compressed_file = os.path.join(args_call.str_out_dir,
+                                                     str_depth_compressed_file)
+            str_depth = " ".join(["samtools", "depth", str_return_bam, ">", str_depth_compressed_file])
+            lcmd_gatk_variants_commands.append(Command(str_depth,'dep.ok'))
+        
+        # Variant calling
+        if args_call.str_gatk_variant_call_method == STR_GATK_HC:
+            str_hap_call = " ".join(["java", "-jar", gatk_path,
+                                     "HaplotypeCaller", "-R",
+                                     genome_fa, "-I", str_return_bam,
+                                     "--recover-dangling-heads","true",
+                                     "--dont-use-soft-clipped-bases","-stand-call-conf",
+                                     "20.0",
+                                     "-O", str_variants_file])
+            lcmd_gatk_variants_commands.append(Command(str_hap_call,'hap_call.ok'))
+          
+        elif args_call.str_gatk_variant_call_method == STR_GATK_MUTECT2 and args_call.str_germline_resource:
+            # if genome_resource is provided
+            f1r2_output = os.path.join(args_call.str_out_dir,"f1r2.tar.gz")
+            gnomad = args_call.str_germline_resource #os.path.join(ctat_genome_lib_path,"ctat_mutation_lib", "af-only-gnomad.raw.sites.b37.vcf.gz")
+
+            str_m2_call = " ".join(["java", "-jar", gatk_path,
+                                     "Mutect2",
+                                     "-R", genome_fa,
+                                     "-I", str_input_bam,
+                                     "--germline-resource", gnomad,
```

Comment:
these are two very different mutect2 calls.  Do we have good justification for keeping them both, and for why one must be very different from the other?  (if so, let's include a comment/link to the gatk url indicating the settings).